clikit/src/main/java/org/adamalang/clikit/exceptions/XMLFormatException.java

```diff
@@ -0,0 +1,23 @@
+package org.adamalang.clikit.exceptions;
+
+import java.util.ArrayList;
+
+public class XMLFormatException extends Exception {
+    /** Exception class used to identify Errors with XML Formatting **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc